buildSrc/src/main/kotlin/io/spine/internal/dependency/ProtoData.kt

```diff
@@ -33,7 +33,7 @@ package io.spine.internal.dependency
  */
 @Suppress("unused", "ConstPropertyName")
 object ProtoData {
-    const val version = "0.9.9"
+    const val version = "0.9.11"
```

Comment:
Just to discuss later. We already have [`0.10.2`](https://github.com/SpineEventEngine/ProtoData/blob/master/version.gradle.kts#L35), and it's currently being updated to in scope of Validation's latest PR.